### PR TITLE
Fix issue #1745 - javascript allow dot in filename

### PR DESCRIPTION
**Pull request by adam-26**

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by adam-26** [comment]
I signed it!

File: src/google/protobuf/compiler/js/js_generator.cc [hunk 1]
```diff
@@ -193,6 +193,7 @@ string ModuleAlias(const string& filename) {
   string basename = StripProto(filename);
   StripString(&basename, "-", '$');
   StripString(&basename, "/", '_');
+  StripString(&basename, “.”, '_');
```
Comment (murgatroid99) [inline src/google/protobuf/compiler/js/js_generator.cc]: Please replace these with normal quotation marks.

Comment (adam-26) [inline src/google/protobuf/compiler/js/js_generator.cc]: Sorry about that, text-editor 'auto-corrected'. Fixed now.

**Comment by acozzette** [comment]
ok to test (this comment will kick off the other CI runs)

**Comment by acozzette** [comment]
@adam-26 There seems to be an issue with the CLA. Could you double-check that you have your GitHub username associated with your CLA? It looks like your email address is hidden on your commits, so the CLA bot can only verify based on your GitHub username.

**Comment by adam-26** [comment]
I signed it!

**Comment by adam-26** [comment]
@acozzette I have updated my CLA settings, sorry about that. Should be OK now.

**Comment by acozzette** [comment]
Great, thanks for the pull request!

**Pull request merged**